pkg/controller.v1/pytorch/pytorchjob_controller.go

```diff
@@ -155,13 +156,18 @@ func (r *PyTorchJobReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	// Set default priorities to pytorch job
 	r.Scheme.Default(pytorchjob)
 
+	// parse volcano Queue from pytorchjob Annotation
```

Comment:
what about other jobs?

Reply:
I did some checks seems MXNet miss this part as well